src/rt/trace.d

```diff
@@ -77,32 +140,9 @@ __gshared
 
     timer_t trace_ohd;
 
-    string trace_logfilename = "trace.log";
-    string trace_deffilename = "trace.def";
```

Comment:
why not just embed a `\0` in them instead?

Reply:
Technically it's already embedded because it's a string literal :)
But the point was that I could not free them, so I'd have to special-case things for the initial case.

Reply:
Ah, fair enough.

Reply:
In particular this wouldn't work if we were pointing to a string literal:
https://github.com/dlang/druntime/pull/2974/files#diff-403c31034f603ad2b3ffcd39aa753a2cR80